micro_dl/scripts/inference_bug_tracing.py

```diff
@@ -0,0 +1,66 @@
+# %%
+import argparse
+import datetime
+import os
+import torch
+import yaml
+import sys
+
+sys.path.insert(0, '/home/christian.foley/virtual_staining/microDL')
+
+import micro_dl.utils.aux_utils as aux_utils
```

Comment:
Yep! I've been using the scripts under microDL/micro_dl/scripts to help with development and I don't intend to include any of them in the final PR. If I've written a script that should be included, it will be stored in microDL/scripts.